pubsub/pstest/fake.go

```diff
@@ -504,6 +504,11 @@ func (s *GServer) CreateSubscription(_ context.Context, ps *pb.Subscription) (*p
 	} else if ps.BigqueryConfig.Table != "" {
 		ps.BigqueryConfig.State = pb.BigQueryConfig_ACTIVE
 	}
+	if ps.CloudStorageConfig == nil {
+		ps.CloudStorageConfig = &pb.CloudStorageConfig{}
```

Comment:
why the empty submessage?  Not populating the message would be my expectation.  This would change some of your expectations elsewhere (and require the typical nil check code for accessing inner fields).